Library/MessageGenerator.swift

```diff
@@ -6,18 +6,15 @@ struct MessageGenerator {
     func message(forDay day: Date, sunPhase: SunPhase, yesterdayDaylightLength: Double, todayDaylightLength: Double, tomorrowDaylightLength: Double) -> Message {
         let defaults = UserDefaults.standard
 
-        let dayKey = self.dateFormatter.string(from: day)
+        let cachingKey = self.cachingKey(forDay: day, andSunPhase: sunPhase)
+        let coloredPartCachingKey = "\(cachingKey)colored"
 
-        if let message = UserDefaults.standard.string(forKey: dayKey), let colored = UserDefaults.standard.string(forKey: "\(dayKey)colored") {
+        if let message = UserDefaults.standard.string(forKey: cachingKey), let colored = UserDefaults.standard.string(forKey: coloredPartCachingKey) {
             return Message(content: message, coloredPart: colored)
         } else {
-            if let appDomain = Bundle.main.bundleIdentifier {
-                UserDefaults.standard.removePersistentDomain(forName: appDomain)
```

Comment:
I removed this since we are using the user defaults for other things and wiping it seems like a bad idea because of it.